### PR TITLE
[RA01 CH02]: Clarifications to multi-threading

### LINKED ISSUE #1701: [RA01 Ch02] Clarifications to multi-threading

### PR DESCRIPTION
**Pull request by EditKoselak**
Closes #1701

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter02.md [hunk 1]
```diff
@@ -70,8 +70,8 @@ Traceability to Reference Model.
...
-| `req.inf.com.09` | Compute | The Architecture **must** ensure that the host cores/threads assigned to a workload are thread-sibling aware: that is, that a core and its associated SMT threads are either all assigned to non-tenant workloads or all assigned to tenant workloads. | Achieved through configuring the "cpu_dedicated_set" and "cpu_shared_set" parameters in nova.conf correctly. |
+| `req.inf.com.08` | Compute | The Architecture **must** support allocating certain number of host cores or SMT threads to non-tenant workloads such as for OpenStack services. | [Dedicating host cores or SMT threads to certain workloads (e.g., OpenStack services](https://docs.openstack.org/nova/latest/configuration/config.html#compute.cpu_dedicated_set)). Please see example, ["Configuring libvirt compute nodes for CPU pinning"](https://docs.openstack.org/nova/latest/admin/cpu-topologies.html) |
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter02.md:73]: OpenStack expects that the cores would be dedicated for these non-tenant workloads; when dedicating a host core its SMT threads are also dedicated to these non-tenant workloads. There is no mechanism to dedicate only SMT threads independent of the host cores.
```suggestion
| `req.inf.com.08` | Compute | The Architecture **must** support allocating certain number of host cores for all non-tenant workloads such as for OpenStack services; SMT threads (vCPUs) can be allocated to individual OpenStack services or their components. | [Dedicating host cores to certain workloads (e.g., OpenStack services](https://docs.openstack.org/nova/latest/configuration/config.html#compute.cpu_dedicated_set)). Please see example, ["Configuring libvirt compute nodes for CPU pinning"](https://docs.openstack.org/nova/latest/admin/cpu-topologies.html) |
```

File: doc/ref_arch/openstack/chapters/chapter02.md [hunk 2]
```diff
@@ -70,8 +70,8 @@ Traceability to Reference Model.
...
+| `req.inf.com.09` | Compute | The Architecture **must** ensure that the host cores or SMT threads assigned to a workload are SMT aware: that is, that a core and its associated SMT threads are either all assigned to non-tenant workloads or all assigned to tenant workloads. | Achieved through configuring the "cpu_dedicated_set" and "cpu_shared_set" parameters in nova.conf correctly. |
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter02.md:74]:
```suggestion
| `req.inf.com.09` | Compute | The Architecture **must** ensure that the host cores assigned to non-tenant and tenant workloads are SMT aware: that is, a host core and its associated SMT threads are either all assigned to non-tenant workloads or all assigned to tenant workloads. | Achieved through configuring the "cpu_dedicated_set" and "cpu_shared_set" parameters in nova.conf correctly. |
```

Comment (EditKoselak) [reply doc/ref_arch/openstack/chapters/chapter02.md]: Thanks, done.

File: doc/ref_arch/openstack/chapters/chapter02.md (hunk 1 above)
Comment (EditKoselak) [reply doc/ref_arch/openstack/chapters/chapter02.md]: Thanks! We have corrected it with minor changes.

**Pull request merged**